pkg/apis/policy/types.go

```diff
@@ -444,3 +449,18 @@ type PodSecurityPolicyList struct {
 
 	Items []PodSecurityPolicy
 }
+
+// RuntimeClassStrategyOptions define the strategy that will dictate the allowable RuntimeClasses
+// for a pod.
+type RuntimeClassStrategyOptions struct {
+	// AllowedRuntimeClasses is a whitelist of RuntimeClass names that may be specified on a pod.
+	// An empty string value allows the RuntimeClassName to be left unset (or set to "").
+	// An empty list allows all RuntimeClasses.
+	// +optional
+	AllowedRuntimeClasses []string
+	// DefaultRuntimeClass is the default RuntimeClassName to set on the pod.
+	// The default MUST be allowed by the above AllowedRuntimeClasses.
+	// A value of nil does not mutate the Pod.
+	// +optional
+	DefaultRuntimeClass *string
```

Comment:
**Question:** Should this be `DefaultRuntimeClassName` to match the pod's `RuntimeClassName` field?

Reply:
yes